### ISSUE TITLE
Cannot mount a custom activemq.xml

### ISSUE THREAD

**Issue by manuelmazzuola** [disaster37/activemq#4]
Hi, if I mount  a custom activemq.xml file on activemq/conf/activemq.xml the init script fails on sed.

`sed: cannot rename conf/sedB1mvOH: Device or resource busy`

The issue is explained here https://docs.docker.com/userguide/dockervolumes/#mount-a-host-file-as-a-data-volume

```
Note: Many tools used to edit files including vi and sed --in-place may result in an inode change. Since Docker v1.1.0, this will produce an error such as "sed: cannot rename ./sedKdJ9Dy: Device or resource busy". In the case where you want to edit the mounted file, it is often easiest to instead mount the parent directory.
```

Then if I mount the parent `conf` directory the init script fails because it can't find your *.properties files (https://github.com/disaster37/activemq/tree/master/assets/config)

**Comment by disaster37** [comment]
It's write ;) I will fix that this week and upgrade ActiveMQ.

**Comment by mikus** [comment]
any news about this issue?

**Comment by vodrazka** [comment]
Still occurs

**Comment by disaster37** [comment]
Ok, I have change the init script. If you try the last dockerfile (not yet ready for production use), I think you can now mount conf directory. I have not yet tested to mount only one config file.

Can you try and confirm that ? Else I will try this weekend ;)

**Comment by disaster37** [comment]
It's fix on release 5.11.1. You get in mind that the environment variable can override your custom setting.
